hardware_interface/src/system.cpp

```diff
@@ -214,6 +214,7 @@ const rclcpp_lifecycle::State & System::get_state() const { return impl_->get_st
 
 return_type System::read(const rclcpp::Time & time, const rclcpp::Duration & period)
 {
+  // TODO(destogl): discuss what should be default return value, e.g., "NOT_EXECUTED"
```

Comment:
ok... I see you added this in multiple places, I take that as a sign of you not wanting to include this login in this PR ;) how about a follow-up issue where we can first agree on what to use as return values (finally we could have more than success and error) and then implement it.